en/activity-standard/iati-activities/iati-activity/default-aid-type.rst

```diff
@@ -2,14 +2,25 @@ Example Usage
 ~~~~~~~~~~~~~
 Example ``default-aid-type``  for an ``iati-activity``.
 
-| The ``@code`` attribute declares a valid code (*A01*) from the *AidType* codelist.
+| The ``@code`` attribute declares a valid code from the specified codelist.
 | The ``@vocabulary`` attribute declares a valid code (*1*) from the *AidTypeVocabulary* codelist.
 
+.. code-block:: xml
+
+	<default-aid-type code="A01" vocabulary="1" />
+
+
+The default-aid-type element can be repeated in any iati-activity.
```

Comment:
I think this should be clarified to something like:

> The ``default-aid-type`` element must not be used more than once for each vocabulary.

---------

[From the discussion on Discuss](https://discuss.iatistandard.org/t/add-vocabularies-to-aid-type-included-2-03/847):
![](https://user-images.githubusercontent.com/961036/45482528-a5b16a80-b74e-11e8-9dbe-d1487c34e7ea.png)

Also, I think this should be done in a new Rule, because at present the (automatically generated) rules on that page are the most confusing (and inaccurate) thing:

![grafik](https://user-images.githubusercontent.com/688113/45830055-94430200-bcfc-11e8-9383-997d88a81737.png)

Reply:
Hi @markbrough your suggested wording, but with a should, is already in the activity schema. We are working on trying to get this shown on the website: https://github.com/IATI/IATI-Schemas/blob/version-2.03/iati-activities-schema.xsd

Reply:
Oh excellent, thanks @amy-silcock! Yes agree that is a better approach :)

Reply:
> We are working on trying to get this shown on the website

I suppose you mean [this line](https://github.com/IATI/IATI-Schemas/blob/a96c582f2f6873aa46b42df2d24d69f3128b24e4/iati-activities-schema.xsd#L2491). Quite difficult to figure out what happened with this. It was added to the wrong branch in IATI/IATI-Schemas#418, then reverted in IATI/IATI-Schemas#420… I guess it was re-added recently, and hasn’t actually been deployed. So I’d suggest deploying v2.03.

As usual, I think auto-deploying the reference site from github would be a really sensible step that could save some headaches. See: IATI/IATI-Standard-SSOT#164.